libqtile/core/manager.py

```diff
@@ -399,6 +399,7 @@ def update_net_desktops(self) -> None:
         # (via current_group), and when we set up the initial groups, there
         # aren't any screens yet. This can probably be changed when #475 is
         # fixed.
+        # #475 is fixed, catching AttributeError is still necessary.
```

Comment:
Ah I see. I misunderstood the first form lol. Thanks for updating it.

Reply:
Thanks for helping me get it right!